src/index.js

```diff
@@ -23,6 +24,7 @@ window.onload = fetchData;
 
 loginSubmitBtn.addEventListener('click', validateForm);
 logOutBtn.addEventListener('click', logOut);
+managerView.addEventListener('click', analyzeManagerClick)
```

Comment:
I think that you could add in all of your event listeners that are listening for the same event into a click handler so that you only need one event listener

Reply:
@ErinUntermeyer I actually had this discussion with my mentor for What's Cookin', and then discussed with Will, because I was also under the impression that we should always only have just 1 event listener for a click. Will said it's not that cut & dried and that you should make use of event delegation to create event listeners for each **section** of your site. Once your site has a lot of clicks, if you create just 1 event listener, your click handler function can become massive & really hard to sort through. So I'm going that route & will have event listeners for the following 4 sections: the header (which is just the log out btn), the login screen, the manager dash, and the customer dash

Reply:
Sounds good!